tox.ini

```diff
@@ -1,8 +1,9 @@
 # Reference: https://github.com/ethereum/ethereum-python-project-template/blob/master/tox.ini
 
-# TODO: consider py36 and pypy3 support
+# TODO: consider pypy3 support
 [tox]
 envlist =
+    py36-test
```

Comment:
nitpick: it seems `py36-test` and `py37-test` can be combined as `py{36,37}-test`